lib/honeybadger/json.ex

```diff
@@ -0,0 +1,53 @@
+defmodule Honeybadger.JSON do
+  def encode(notice) do
```

Comment:
We don't make any use of dialyzer currently, but I've been trying to include type specs when we introduce or update functions. It would be nice to have one here as well.